website/static/resource/studio/documentation/dsl-service.json

```diff
@@ -0,0 +1,99 @@
+{
+    "entries": {
+        "dsl-service.grammar.parser": {
+            "markdownText": {
+                "value": "`Service DSL` (coressponding to `###Service` section in `Pure`) concerns with generation and deployment of Pure query to productionize data extraction and exploration"
+            },
+            "title": "What is Service DSL about?"
+        },
+        "dsl-service.grammar.element.service": {
```

Comment:
unfortunately, I think this is the only entry that is relevant to be in this file, the rest of the entries should be in `studio.json`